demo/test_server.html

```diff
@@ -19,23 +19,35 @@ <h2>fxa-pairing-tls demo server</h2>
 
   <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
   <script src="../dist/fxaPairingTLS.js"></script>
-  <script src="./tlschannel.js"></script>
   <script>
     $(async () => {
+      // TODO: do we want to export InsecurePairingChannel directly?
```

Comment:
we can export that as `_util` like we do in https://github.com/mozilla/fxa-crypto-relier/blob/master/src/relier/OAuthUtils.js#L157